### ISSUE TITLE
Geckodriver Failed to Start Browser - Other OS Error in Windows Docker Container

### ISSUE THREAD

**Issue by BainsDev** [mozilla/geckodriver#1734]
## System

* Version: 0.26
* Platform: Docker Container for Windows (1909 full base image) and 1909 host OS
* Firefox: 77
* Selenium: 3

## Testcase

```
            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
            var options = new FirefoxOptions();
            options.Profile = profile;
            options.AddArgument("--headless");
            FirefoxDriverService service = FirefoxDriverService.CreateDefaultService();
            service.Host = "::1";
            return new FirefoxDriver(service, options);

```

## Stacktrace
```
1592252364255  mozrunner::runner   INFO    Running command: "C:\\Program Files\\Mozilla Firefox\\firefox.exe" "-marionette" "--headless" "-foreground" "-no-remote" "-profile" "C:\\Users\\ContainerAdministrator\\AppData\\Local\\Temp\\rust_mozprofileLoSBip"
    Exception thrown: 'System.InvalidOperationException' in WebDriver.dll
    [06/15/2020 13:16:10, Error]: System.InvalidOperationException: Failed to start browser C:\Program Files\Mozilla Firefox\firefox.exe: other os error (SessionNotCreated)
       at OpenQA.Selenium.Remote.RemoteWebDriver.UnpackAndThrowOnError(Response errorResponse)
       at OpenQA.Selenium.Remote.RemoteWebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)
       at OpenQA.Selenium.Remote.RemoteWebDriver.StartSession(ICapabilities desiredCapabilities)
       at OpenQA.Selenium.Remote.RemoteWebDriver..ctor(ICommandExecutor commandExecutor, ICapabilities desiredCapabilities)
       at OpenQA.Selenium.Firefox.FirefoxDriver..ctor(FirefoxDriverService service, FirefoxOptions options, TimeSpan commandTimeout)
       at OpenQA.Selenium.Firefox.FirefoxDriver..ctor(FirefoxDriverService service, FirefoxOptions options)

```

I'm trying to run Firefox headless with GeckoDriver in my Docker container for Windows. I've made sure I have .Net Framework installed, along with the VC Redist (which apparently resolved this issue for some people on Windows OSes), however i'm still facing this issue in my Docker container. I tried Firefox Portable as well and it gives the exactly same error. Also tried changing my profile directory but no dice.

**Comment by whimboo** [comment]
Which version of Windows 10 is it? The 32bit or 64bit one? Did you install the corresponding Firefox version (32 vs 64 bit)? Note that both are getting installed in different paths. 

Most likely this is a dupe of #940.

**Comment by Zven214** [comment]
Hi there from the land of Debian!
I had a similar issue with "other os error" after I had been very busy testing things, it turns out that error message is a catch all for "it was odd". I had been filling up a tmp directory with bits and pieces of all the sessions I had ran and closed feverishly, once it was filled the problem occurred and once I restarted (which cleans that directory) It worked again, and I was a bit wiser.

Hope this helps.
